build-support/scripts/up

```diff
@@ -12,15 +12,15 @@ docker run -d --name ad1 --rm \
     -p 48005:48005 \
     -e "NUODB_DOMAIN_ENTRYPOINT=ad1" \
     ${IMG_NAME} nuoadmin
+#changed to properly create and mount volumes
+docker volume create vol1
+
+docker volume create vol2
```

Comment:
Why don't you call these `backup` and `archive`, since that is what they are used for?

Reply:
That would be a good change. I stuck with the names that were in here already.